packages/backend/src/open_payments/payment/incoming/routes.ts

```diff
@@ -149,19 +149,23 @@ async function listIncomingPayments(
     await listSubresource({
       ctx,
       getPaymentPointerPage: deps.incomingPaymentService.getPaymentPointerPage,
-      toBody: (payment) =>
-        incomingPaymentToBody(payment, deps.connectionService.getUrl(payment))
+      toBody: async (payment) =>
+        await incomingPaymentToBody(
+          payment,
+          deps.connectionService.getUrl(payment)
```

Comment:
Could we either make this `getUrl` async for consistency or rename it?

Reply:
I _think_ it should be ok as it is, since it's a service method, not a model?